### PR TITLE
Fix #3: upgrade to 11.1

### LINKED ISSUE #3: Upgrade to cuda-11-1
cuda repos for fedora 32 do not support 10-1 https://github.com/radiasoft/download/issues/131

### PR DESCRIPTION
**Pull request by e-carlin**
I built the image on the target server and tried two tests. Lmk if you have better ways of testing. 

1. Ran `ffmpeg -y -vsync 0 -hwaccel cuda -hwaccel_output_format cuda -i input.mp4 -c:a copy -c:v h264_nvenc -b:v 5M output.mp4`
2. I also installed cuda-samples-11-1-11.1.74-1.x86_64 and ran the "simple" sample. The sample seemed to run fine but there was no "success" message or anything like that so I'm not 100% positive it worked.

### CODE REVIEW

**Comment by e-carlin** [comment]
I've also added some development notes https://github.com/radiasoft/container-jupyter-nvidia/wiki/Development

File: container-conf/build.sh [hunk 1]
```diff
@@ -10,23 +10,21 @@ build_is_public=1
...
 build_as_root() {
     umask 022
     cd "$build_guest_conf"
-    dnf config-manager --add-repo http://developer.download.nvidia.com/compute/cuda/repos/fedora29/x86_64/cuda-fedora29.repo
-    build_yum install cuda-toolkit-10-1
+    dnf config-manager --add-repo http://developer.download.nvidia.com/compute/cuda/repos/fedora32/x86_64/cuda-fedora32.repo
+    build_yum install cuda-toolkit-11-1
     cat > /etc/ld.so.conf.d/rs-cuda.conf <<'EOF'
-/usr/local/cuda-10.2/targets/x86_64-linux/lib
-/usr/local/cuda/extras/CUPTI/lib64
```
Comment (robnagler) [inline container-conf/build.sh:24]: Are you sure CUPTI is not required/installed? I see that it is still being developed.

File: container-conf/build.sh [hunk 2]
```diff
@@ -10,23 +10,21 @@ build_is_public=1
...
+/usr/local/cuda-11.1/targets/x86_64-linux/lib
 EOF
-    # https://gitlab.com/nvidia/container-images/cuda/blob/master/dist/centos7/10.1/runtime/cudnn7/Dockerfile
-    # modified to use 10.0
-    build_curl -O http://developer.download.nvidia.com/compute/redist/cudnn/v7.6.5/cudnn-10.1-linux-x64-v7.6.5.32.tgz
-    tar --no-same-owner -xzf cudnn-10.1-linux-x64-v7.6.5.32.tgz -C /usr/local --wildcards 'cuda/lib64/libcudnn.so.*'
+    # https://gitlab.com/nvidia/container-images/cuda/-/blob/master/dist/11.1.1/centos7-x86_64/runtime/cudnn8/Dockerfile
+    dnf config-manager --add-repo https://developer.download.nvidia.com/compute/cuda/repos/rhel7/x86_64/cuda-rhel7.repo
+    build_yum install libcudnn8-8.0.5.39-1.cuda11.1.x86_64
```
Comment (robnagler) [inline container-conf/build.sh:27]: does this need ldconfig or is in the rpm?

File: container-conf/build.sh [hunk 3]
```diff
@@ -10,23 +10,21 @@ build_is_public=1
...
     umask 022
     cd "$build_guest_conf"
-    dnf config-manager --add-repo http://developer.download.nvidia.com/compute/cuda/repos/fedora29/x86_64/cuda-fedora29.repo
-    build_yum install cuda-toolkit-10-1
+    dnf config-manager --add-repo http://developer.download.nvidia.com/compute/cuda/repos/fedora32/x86_64/cuda-fedora32.repo
+    build_yum install cuda-toolkit-11-1
     cat > /etc/ld.so.conf.d/rs-cuda.conf <<'EOF'
-/usr/local/cuda-10.2/targets/x86_64-linux/lib
-/usr/local/cuda/extras/CUPTI/lib64
+/usr/local/cuda-11.1/targets/x86_64-linux/lib
```
Comment (robnagler) [inline container-conf/build.sh:23]: Is this not in the rpm still? Probably not, just making sure.

**Review by robnagler — APPROVED** [review]

File: container-conf/build.sh (hunk 1 above)
Comment (e-carlin) [reply container-conf/build.sh:24]: The CUPTI libs now seem to be in /usr/local/cuda-11.1/targets/x86_64-linux/lib so just that line covers everything.

File: container-conf/build.sh (hunk 3 above)
Comment (e-carlin) [reply container-conf/build.sh:23]: I didn't test. https://github.com/radiasoft/container-jupyter-nvidia/issues/5

File: container-conf/build.sh (hunk 2 above)
Comment (e-carlin) [reply container-conf/build.sh:27]: I thought the ldconfig was for adding /etc/ld.so.conf.d/rs-cuda.conf ?

Comment (robnagler) [reply container-conf/build.sh:27]: Yes, I meant what you wrote in #5. Sorry for not being clear.

**Pull request merged**